autoload/maktaba.vim

```diff
@@ -105,3 +110,22 @@ function! maktaba#LateLoad(...) abort
     call maktaba#filetype#Cycle()
   endif
 endfunction
+
+
+""
+" @private
+" Forces the disabling of the Python implementation of the maktaba#json
+" functions, to enable testing. This must be called before referencing any of
+" the maktaba#json functions, since it will only take effect on first load.
+function! maktaba#SetJsonPythonDisabled(disabled) abort
```

Comment:
Actually, I've just spotted a bigger problem: calling this `maktaba#something` means that we'll trigger plugin installation (from `autoload/maktaba.vim`) before we have a chance to check the flag. I don't know why I didn't spot that as a problem earlier.